src/DateUtils.ts

```diff
@@ -123,6 +123,19 @@ export function formatTime(date: Date, showTwelveHour = false): string {
     return pad(date.getHours()) + ':' + pad(date.getMinutes());
 }
 
+export function formatCallTime(date: Date): string {
```

Comment:
`date` -> `delta` maybe, or something else that doesn't scream an absolute value

Reply:
Yeah, makes senes :+1: